transcom/mymove
# INT B-21117 boat shipment db changes

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21117)

## Summary

Adds a database migration to update the pricing factors for boat tow away and haul away shipments.

## How to Test

Run `make db_dev_fresh` then access the DB using DBeaver or psql. Query the `re_shipment_type_prices` table, and ensure the two records whose `service_id` foreign keys point to the tow away and haul away boat types in the `re_services` table have had their factors updated.

The haul away type should have a factor of 45.77 while the tow away should have a factor of 35.33
